src/resolve/resolved-conf.c

```diff
@@ -413,3 +413,45 @@ int manager_parse_config_file(Manager *m) {
 #endif
         return 0;
 }
+
+int config_parse_record_types(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+        int r;
+
+        if (isempty(rvalue)) {
+                m->refuse_record_types = set_free(m->refuse_record_types);
+                return 0;
+        }
+
+        for (const char *p = rvalue;;) {
+               _cleanup_free_ char *word = NULL;
+               r = extract_first_word(&p, &word, NULL, 0);
+               if (r < 0)
+                     return log_syntax_parse_error(unit, filename, line, r, lvalue, rvalue);
+               if (r == 0)
+                     return 1;
```

Comment:
our indentation is 8ch, this appears to be 6ch?

Reply:
Yeah, the `if` function line also lacked one more space. I've fixed all (hopefully)